### ISSUE TITLE
Add pitch accent markers where furigana is being displayed

### ISSUE THREAD

**Issue by Morgawr** [themoeway/yomitan#294]
Not all entries may have pitch accent info, and there may be more than one entry or different pitch accent dictionaries, but I think in general it would be really cool to be able to provide some pitch accent markers visible on the furigana itself. 

For example take this entry from a random VN I am playing: 
![image](https://github.com/themoeway/yomitan/assets/184056/b6c23f57-da72-478b-a203-38c2f94358fb)

I think it would be cool to have something like pic related instead (100 hours in MS Paint btw):
![image](https://github.com/themoeway/yomitan/assets/184056/f447f3b3-ef77-44be-a313-140fade1dbf9)

So basically:
- Add pitch markers on furigana for individual words in the search query
- Add pitch markers on the actual furigana of the word looked up in the popup itself (the 頭上 entry in this case)
- (optional) Add pitch markers on the non-furigana kana words like そして or から in my example

Obviously with the caveat that pitch is incredibly complicated and that sentence-level pitch and combinations (like XのY, etc) may override the individual vocab rules for each entry and that users should be aware of that pitfall.
